validator_manager/src/exit_validators.rs

```diff
@@ -0,0 +1,576 @@
+                    "Beacon node is syncing, submit the voluntary exit later when beacon node is synced"
+                        .to_string(),
+                );
+            }
+
+            let genesis_data = beacon_node
+                .get_beacon_genesis()
+                .await
+                .map_err(|e| format!("Failed to get genesis data: {}", e))?
+                .data;
+
+            let config_and_preset = beacon_node
+                .get_config_spec::<ConfigAndPreset>()
+                .await
+                .map_err(|e| format!("Failed to get config spec: {}", e))?
+                .data;
+
+            let spec = ChainSpec::from_config::<E>(config_and_preset.config())
+                .ok_or("Failed to create chain spec")?;
+
+            let validator_data = beacon_node
+            .get_beacon_states_validator_id(StateId::Head, &ValidatorId::PublicKey(validator_to_exit))
+            .await
+            .map_err(|e| format!("Failed to get validator details: {:?}", e))?
+            .ok_or_else(|| {
+                format!(
+                    "Validator {} is not present in the beacon state. \
+                Please ensure that your beacon node is synced and the validator has been deposited.",
+                    validator_to_exit
+                )
+            })?
+            .data;
+
+            let activation_epoch = validator_data.validator.activation_epoch;
+            let current_epoch = get_current_epoch::<E>(genesis_data.genesis_time, &spec)
+                .ok_or("Failed to get current epoch. Please check your system time")?;
+
+            // Check if validator is eligible for exit
+            if validator_data.status == ValidatorStatus::ActiveOngoing
+                && current_epoch < activation_epoch + spec.shard_committee_period
+            {
+                eprintln!(
+                    "Validator {} is not eligible for exit. It will become eligible at epoch {}",
+                    validator_to_exit,
+                    activation_epoch + spec.shard_committee_period
+                )
+            } else if validator_data.status != ValidatorStatus::ActiveOngoing {
+                eprintln!(
+                    "Validator {} is not eligible for exit. Validator status is: {:?}",
+                    validator_to_exit, validator_data.status
+                )
+            } else {
+                // Only publish voluntary exit if validator status is ActiveOngoing
+                beacon_node
+                    .post_beacon_pool_voluntary_exits(&exit_message.data)
+                    .await
+                    .map_err(|e| format!("Failed to publish voluntary exit: {}", e))?;
+                eprintln!(
+                    "Successfully validated and published voluntary exit for validator {}",
+                    validator_to_exit
+                );
+            }
+
+            // Check validator status after publishing voluntary exit
+            if exit_status {
```

Comment:
There are quite some flags being introduced here: `--validators`, `--beacon-node`, `--signature`, `--exit-epoch`, `--status`. The first four are required, but I am not sure about the last one:  `--status`, which is used to query the exit status of validators. 

The reason behind introducing it is:

In `lighthouse account validator exit`, the status was shown after a sleep of, e.g., 12s

https://github.com/sigp/lighthouse/blob/0d90135047519f4c2ee586d50e560f7bb2ff9b10/account_manager/src/validator/exit.rs#L204

If we introduce a sleep here, then for each `validator_to_exit`, it will sleep for 12s, which is not really UX friendly. For practical consideration, the user's priority may just want to first publish the exit messages for all validators.

While there is a way to tweak it so that it only sleeps once and not for the rest of `validator_to_exit` (or shortening the sleep time), doing so will likely cause the exit status to be shown as:

`Voluntary exit for Validator {} is waiting to be accepted into the beacon chain.`

for the rest of `validator_to_exit` as the head block is not yet updated, which is not really useful either.

So I ended up introducing this `--status` flag to query the exit status separately. But I am not sure if this is a good idea. 

Keen on hear the thoughts of reviewers on this.